README.md

```diff
@@ -1,2 +1,45 @@
-# geo-export
-export shapes
+GeoExport
+----
+
+## Testing
+You shouldn't need anything fancy to test. Just run `sbt test`
+
+## Running
+
+#### Dependencies
+* You need to have `zookeeper` running.
+* You need to have `soda-fountain` running.
+* `soda-fountain` must be registered in `zookeeper` and responding to requests
```

Comment:
Have you had problems with this?  "responding to requests" seems somewhat implicit within "running".